### DISCUSSION TITLE
Welcome to JanusGraph Discussions!

### DISCUSSION THREAD

**Discussion by porunov** [JanusGraph/janusgraph#2278] [General]
## 👋 Welcome!
**JanusGraph Discussions is for questions about using JanusGraph, installation, configuration, integrations and troubleshooting.**

In addition to GitHub Discussions, you can post your questions on [janusgraph-users mailing list](https://lists.lfaidata.foundation/g/janusgraph-users), [StackOverflow](https://stackoverflow.com/questions/tagged/janusgraph), [Discord](https://discord.gg/5n4fjv4QAf) and [Gitter](https://gitter.im/janusgraph/janusgraph).

Notice, questions regarding the Gremlin graph traversal language in general should be raised in the [Gremlin users group](https://groups.google.com/forum/#!forum/gremlin-users) or [Gremlin Discord server](https://discord.gg/ndMpKZcBEE).

**Comment by phreed** [comment]
I think the main Gremlin discussions now happen on Discord.
https://discord.com/invite/ndMpKZcBEE
see https://groups.google.com/g/gremlin-users/c/CR2-nVrx4WE

Did I see a discussion about having a discord channel for JanusGraph?

**Reply by li-boxuan** [reply]
> Did I see a discussion about having a discord channel for JanusGraph?

Yes, in the TinkerPop Discord channel, @FlorianHockmann proposed creating a discord channel for JanusGraph.

AFAIK, the only downside of using Discord is that the chat history is not searchable on Google.

**Reply by FlorianHockmann** [reply]
Sorry for the long delay, but I started a discussion [on our dev list](https://lists.lfaidata.foundation/g/janusgraph-dev/message/1568) about migrating to Discord and now also on janusgraph-users: https://lists.lfaidata.foundation/g/janusgraph-users/message/6500

So, anyone interested in this can voice their opinion there.

**Reply by FlorianHockmann** [reply]
The Discord Server is now ready: https://discord.gg/5n4fjv4QAf
Everyone interested in JanusGraph is invited to join 🎉

**Comment by MounikaMandi** [comment]
Hi,
The index creation for large graph data set is taking very much time in janus graph.More over , when ever we need to create a new index we are stopping all the ingestion until index creation is completed.
I wanted to know is there a easy way to process index creation without stopping ingestion?

Thanks

**Reply by farodin91** [reply]
Could please open an extra thread? This is just description.
